### ISSUE TITLE
[Code-Transform] phpactor does not import a class if called from a docblock

### ISSUE THREAD

**Issue by ek-yrabiller** [phpactor/phpactor#1129]
I could not find any issue regarding this problem, but maybe I missed it.

Given this file:

```php
<?php

namespace App\Cognito;

use Aws\CognitoIdentityProvider\CognitoIdentityProviderClient;

class Migrator
{
    private CognitoIdentityProviderClient $cognitoClient;

    /** @var Gro<>upMigrator[] */
    private array $groupMigrators;
}
```

If I try to import the class, I get the message `Imported class "App\Cognito\Migrator\GroupMigrator"` but nothing changed.

On the other hand, if I add the class name everywhere not in a comment:

```php
<?php

namespace App\Cognito;

use Aws\CognitoIdentityProvider\CognitoIdentityProviderClient;

class Migrator
{
    private CognitoIdentityProviderClient $cognitoClient;

    /** @var GroupMigrator[] */
    private array $groupMigrators;
    Gro<>upMigrator
}
```

It works:

```php
<?php

namespace App\Cognito;

use App\Cognito\Migrator\GroupMigrator;
use Aws\CognitoIdentityProvider\CognitoIdentityProviderClient;

class Migrator
{
    private CognitoIdentityProviderClient $cognitoClient;

    /** @var GroupMigrator[] */
    private array $groupMigrators;
    GroupMigrator
}
```

**Comment by dantleech** [comment]
I thought @elythyr fixed this recently? works for me in `develop`

**Comment by elythyr** [comment]
Hi,

There was indeed another issue for that: https://github.com/phpactor/phpactor/issues/1106

It was fixed on develop, I just checked it works for me.
For anyone who might come by I wan to precise that it only works when importing the class using the vim plugin, like the command `PhpactorImportClass` for example.
Currently we don't autocomplete in the phpdoc so we can't benefit from the import on completion feature.

It would work but only for annotations, like `@NotBla<>` for the `NotBlank` validator with Symfony.
